src/test/kotlin/domain/ScoreTest.kt

```diff
@@ -0,0 +1,56 @@
+package domain
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class ScoreTest {
+    @Test
+    fun `점수가 22이면 버스트이다`() {
+        // given
+        val score = Score.valueOf(22)
+
+        // when
+        val actual = score.isBurst()
+
+        // then
+        assertThat(actual).isEqualTo(true)
+    }
+
+    @Test
+    fun `점수가 21이면 버스트가 아니다`() {
```

Comment:
```suggestion
    fun `점수가 21이면 블랙잭이다`() {
```
21은 블랙잭을 상징하니, 버스트가 아니다라는 것보다는 블랙잭임을 나타내는 것이 더 좋을 것 같아요.